pkg/webservice/handlers.go

```diff
@@ -59,6 +60,7 @@ const (
 	GroupNameMissing         = "Group name is missing"
 	ApplicationDoesNotExists = "Application not found"
 	NodeDoesNotExists        = "Node not found"
+	UnsupportedCompType      = "Compression type not support"
```

Comment:
nit: "not supported"

Reply:
This shouldn't be here at all. As I've repeatedly indicated we should never fail with an error, just fall back to no compression.